src/main/java/com/bobocode/hoverla/bring/annotation/Inject.java

```diff
@@ -0,0 +1,14 @@
+package com.bobocode.hoverla.bring.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation used to indicate that the value should be injected by IoC container
+ */
+@Target({ElementType.FIELD, ElementType.CONSTRUCTOR})
```

Comment:
@DanielChanger i think it’s okay to have it on constructor as well. Anyway we will have a single constructor, and in case it is a plain one with parameters, it will be used as if it was annotated with @Inject.
Moreover it is more user-friendly to allow this annotation on constructor